<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterEditorDialog</class>
 <widget class="QDialog" name="FilterEditorDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced Filter</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="newBtn">
       <property name="toolTip">
        <string>Clear Filter Tree</string>
       </property>
       <property name="text">
        <string>New</string>
       </property>
       <property name="icon">
        <iconset resource="mtgcards.qrc">
         <normaloff>:/resources/icons/blank33.svg</normaloff>:/resources/icons/blank33.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="openBtn">
       <property name="toolTip">
        <string>Open Filter Tree from file ...</string>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset resource="mtgcards.qrc">
         <normaloff>:/resources/icons/folder215.svg</normaloff>:/resources/icons/folder215.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveBtn">
       <property name="toolTip">
        <string>Save Filter Tree to file ...</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="mtgcards.qrc">
         <normaloff>:/resources/icons/save20.svg</normaloff>:/resources/icons/save20.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="addGroupBtn">
       <property name="toolTip">
        <string>Add Group</string>
       </property>
       <property name="text">
        <string>Add Group</string>
       </property>
       <property name="icon">
        <iconset resource="mtgcards.qrc">
         <normaloff>:/resources/icons/squares40.svg</normaloff>:/resources/icons/squares40.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addFilterBtn">
       <property name="toolTip">
        <string>Add Filter</string>
       </property>
       <property name="text">
        <string>Add Filter</string>
       </property>
       <property name="icon">
        <iconset resource="mtgcards.qrc">
         <normaloff>:/resources/icons/add186.svg</normaloff>:/resources/icons/add186.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteNodeBtn">
       <property name="toolTip">
        <string>Remove selected Filter or Group</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="mtgcards.qrc">
         <normaloff>:/resources/icons/clear5.svg</normaloff>:/resources/icons/clear5.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mtgcards.qrc"/>
 </resources>
 <connections/>
</ui>
